Packages/Netherlands3D/T3DPipeline/Runtime/Scripts/ParseCityJSON/CityJSONStructure/CityJSON.cs

```diff
@@ -203,7 +213,7 @@ private void SetRelativeCenter()
             {
                 var relativeCenterRD = (MinExtent + MaxExtent) / 2;
                 Debug.Log("Setting Relative RD Center to: " + relativeCenterRD);
-                CoordConvert.zeroGroundLevelY = (float)relativeCenterRD.z;
+                CoordConvert.zeroGroundLevelY = 0;// (float)relativeCenterRD.z;
```

Comment:
Wellicht commentaar bij doen..

Reply:
Sam gaat hier 2.5 weken over nadenken en dan kunnen we een betere oplossing maken